charts/routes/templates/mlp-oathkeeper-rules.yaml

```diff
@@ -0,0 +1,68 @@
+{{- if and .Values.oathkeeperRules.enabled .Values.mlp.enabled }}
```

Comment:
i see rules added for mlp, turing and xp., what about merlin? do we not need it?

Reply:
Not added for Merlin at the moment, because we can't delegate the authorization workflow to Oathkeeper for Merlin until the model endpoints are refactored to go under /projects .